runtime/lua/vim/treesitter.lua

```diff
@@ -326,10 +326,21 @@ function M.get_captures_at_cursor(winnr)
   return captures
 end
 
---- @class vim.treesitter.GetNodeOpts
+--- Optional keyword arguments:
+--- @class vim.treesitter.get_node.Opts
+--- @inlinedoc
+---
+--- Buffer number (nil or 0 for current buffer)
 --- @field bufnr integer?
+---
+--- 0-indexed (row, col) tuple. Defaults to cursor position in the
+--- current window. Required if {bufnr} is not the current buffer
 --- @field pos { [1]: integer, [2]: integer }?
+---
+--- Parser language. (default: from buffer filetype)
 --- @field lang string?
```

Comment:
Does `@field` require the docstring to live above it, or can it work like `@param` ?

Reply:
I believe LuaLS requires it above. Same applies to the `@class` description.

Reply:
![image](https://github.com/neovim/neovim/assets/7904185/cdbcd949-7540-4e50-aa56-87258272436f)